web_responsive/readme/DESCRIPTION.rst

```diff
@@ -55,3 +55,7 @@ Features for computers:
 * Full width form sheets
 
   .. image:: https://user-images.githubusercontent.com/973709/48417428-ac5a5f00-e74a-11e8-8839-5bc538c54c1d.png
+
+* Sticky chatter topbar
+
+  .. image:: ./images/chatter_topbar.gif
```

Comment:
May I suggest you to link the gif from Github's CDN instead? It will be rendered everywhere and reduce repo size. (Sometimes relative links are interpreted differently on odoo itself, on the apps store, on github, etc.)

Also, remove the gif file from the PR.

```suggestion
  .. image:: https://user-images.githubusercontent.com/973709/55539179-f8854c00-56b7-11e9-9dc5-1a060123edf3.gif
```

It's using the same URL Github assigns when drag & drop this image here:

![chatter_topbar](https://user-images.githubusercontent.com/973709/55539179-f8854c00-56b7-11e9-9dc5-1a060123edf3.gif)